src/components/themed-image/index.tsx

```diff
@@ -0,0 +1,31 @@
+import { ThemedImageProps } from './types'
+
+/**
+ * This component can be used in views where we need to
+ * conditionally render an image based on the theme state.
+ *
+ * See styles/themes/global.css for the attribute handler
+ * that uses `display:none` to conditionally render content
+ * based on the theme state with css
+ */
+export default function ThemedImage({ src, alt }: ThemedImageProps) {
```

Comment:
Feels pretty simple but works well in the browser. Open to feedback!

Reply:
Ideally we would use next image here but id be nervous about making that work for all contexts. I could also localize this to the product landing for now until we have another use case.

Reply:
I ended up localizing this to the product landing dir for now since this is the only known usecase and I can't anticipate all of the global component's interface for this initial PR.

Reply:
Have we tried using the picture element here? Now that we're on next-13 with the updated next/image, I think this pattern works (haven't fully tested). And removes the need for the CSS hide/show based on theme value.

```tsx
import Image, { type ImageProps } from 'next/image'

interface ThemedImageProps extends ImageProps {
  darkSrc: string
}

function ThemedImage(props: ThemedImageProps) {
  const { darkSrc, ...restProps } = props
  return (
    <picture>
      <source srcSet={darkSrc} media="(prefers-color-scheme: dark)" />
      <Image {...restProps} />
    </picture>
  )
}
```

Reply:
Do we intend to support themes outside of dark and light?

Reply:
ah right right

Reply:
@BRKalow good call, somehow that was on my TODOs but i'll refactor this to use `next/image` and consider the slot pattern you mentioned. If not in this PR, I think a version with slots will be helpful when we lift this up globally.

Reply:
@alexcarpenter that is a super good tip about the `picture` element. i'm gonna look into that even if it doesn't have the flexibility for additional theming. I'd say the likelihood of us creating additional themes is high.

Reply:
Oh @BRKalow now I remember, the issue with using next-image is that all of these graphics have a different size. I could try to fiddle with the styles where you don't input the size 🤔

Reply:
Ooh yeah, makes sense. Feel free to punt on this if it feels like too much of a time sink then 👍

Reply:
Okay, after looking into it, they all mostly had the same height, so was able to refactor to use next/image! Let me know what you think @BRKalow